frontend/Movie_Pigeon/app/src/main/java/org/example/team_pigeon/movie_pigeon/CinemaScheduleFragment.java

```diff
@@ -0,0 +1,235 @@
+import java.util.Calendar;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.Response;
+
+/**
+ * Created by Guo Mingxuan on 23/3/2017.
+ */
+
+public class CinemaScheduleFragment extends Fragment {
+    private ArrayList<Movie> movieList = new ArrayList<>();
+    private ArrayList<ArrayList<Movie>> weekList = new ArrayList<>();
+    private TimeUtil timeUtil = new TimeUtil();
+    private List<Date> dateList;
+    private List<String> dateListString;
+    private ListView scheduleListView;
+    private RadioGroup dateGroup;
+    private RadioButton[] dateButtons = new RadioButton[7];
+    private NowShowingListAdapter adapter;
+    private String TAG = "CinemaScheduleFragment";
+    private Map<Integer, Integer> dateIdMap = new HashMap<>(7);
+    private RequestHttpBuilderSingleton httpRequestBuilder = RequestHttpBuilderSingleton.getInstance();
+    private Gson gson = new Gson();
+    private String cinemaId;
+    private GlobalReceiver receiver;
+    private final int weekListUpdated = 1;
+
+    @Override
+    public void onCreate(Bundle savedInstanceState){
+        super.onCreate(savedInstanceState);
+        Bundle bundle = getArguments();
+        cinemaId = bundle.getString("cinemaId");
+        dateList = timeUtil.getDateList();
+        dateListString = timeUtil.getDateListToString_YYYYMMDD(dateList);
+        new getMovieWorkingThread().execute();
+    }
+
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
+        View view = inflater.inflate(R.layout.fragment_cinema_schedule, container, false);
+        scheduleListView = (ListView) view.findViewById(R.id.cinema_movies_list);
+        dateGroup = (RadioGroup) view.findViewById(R.id.rg_dates);
+        for(int i = 0;i<7;i++){
+            String dbId = "rb_date_" + i;
+            int resID = getResources().getIdentifier(dbId, "id", "org.example.team_pigeon.movie_pigeon");
+            dateButtons[i] = (RadioButton) view.findViewById(resID);
+            dateButtons[i].setText(dateListString.get(i).substring(5));
+            dateIdMap.put(resID,i);
+        }
+        dateGroup.setOnCheckedChangeListener(new RadioGroup.OnCheckedChangeListener()
+        {
+            public void onCheckedChanged(RadioGroup group, int checkedId)
+            {
+                RadioButton checkedRadioButton = (RadioButton)group.findViewById(checkedId);
+                boolean isChecked = checkedRadioButton.isChecked();
+                if (isChecked) {
+                    Log.d(TAG, "onCheckedChanged: button clicked, refreshing adapter");
+                    adapter = new NowShowingListAdapter(weekList.get(dateIdMap.get(checkedId)), getActivity());
+                    scheduleListView.setAdapter(adapter);
+                    ((NowShowingListAdapter)scheduleListView.getAdapter()).notifyDataSetChanged();
```

Comment:
this was actually used when I was debugging to see if the adapter is correctly set. You are right it is no longer needed